code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -229,6 +229,75 @@ Contains:
 	species_exception = list(/datum/species/angel)
 	tauric = TRUE		//Citadel Add for tauric hardsuits
 
+	//Red alert ERT
+
+/obj/item/clothing/head/helmet/space/hardsuit/ert/alert
+	name = "emergency response unit helmet"
+	desc = "Red alert command helmet for the ERT. This one is more armored than its standard version."
+	icon_state = "hardsuit0-ert_commander-alert"
+	item_state = "hardsuit0-ert_commander-alert"
+	item_color = "ert_commander-alert"
+	armor = list("melee" = 75, "bullet" = 60, "laser" = 60, "energy" = 60, "bomb" = 60, "bio" = 100, "rad" = 100, "fire" = 90, "acid" = 80)

```

Comment:
Not even death armor gives that much laser armor.
I also think they should be fire proof or such